case-lib/logging_ctl.sh

```diff
@@ -28,9 +28,16 @@ _func_log_cmd()
     # writing functions
     shopt -s expand_aliases
     if [ "X1" ]; then
+        local ppcmd=$(ps -p $PPID -o cmd --noheader|awk '{print $2;}') ext_message=""
+        [[ "$(file $ppcmd 2>/dev/null |grep 'shell script')" ]] && ext_message=" Sub-Test:"
```

Comment:
I think sometimes it is not the sub-test even cmd name is not the $0. For example, the output from lib.sh. This should not a sub-test.

Reply:
compare with $0 because I want to use it with this pull request: https://github.com/thesofproject/sof-test/pull/5

So when trigger this function, it will load itself to avoid test case stop when lost ssh connect